### PR TITLE
sync dbt metadata to metabase

### PR DESCRIPTION
**Pull request by atvaccaro**
Uses the [Jarvus fork of dbt-metabase](https://github.com/JarvusInnovations/dbt-metabase) to sync dbt metadata (i.e. table descriptions, column descriptions, and column semantic types) to our Metabase instance.

~There is a small TODO to support doc blocks, which will not be hard.~ Nevermind, I'm just bad at Docker.

[An example:](https://dashboards.calitp.org/admin/datamodel/database/2/table/329)
image

I've tested this via docker-compose.
```
[2022-04-11 21:33:43,776] {pod_launcher.py:149} INFO - INFO     :+1: Field GTFS_RT_FACT_FILES.HOUR_EXTRACTED is           metabase.py:416
[2022-04-11 21:33:43,778] {pod_launcher.py:149} INFO -          up-to-date
[2022-04-11 21:33:43,803] {pod_launcher.py:149} INFO - INFO     :+1: Field GTFS_RT_FACT_FILES.MINUTE_EXTRACTED is         metabase.py:416
[2022-04-11 21:33:43,804] {pod_launcher.py:149} INFO -          up-to-date
[2022-04-11 21:33:43,805] {pod_launcher.py:149} INFO - INFO     :fast_forward: Skipping                                             metabase.py:208
[2022-04-11 21:33:43,805] {pod_launcher.py:149} INFO -          source.calitp_warehouse.gtfs_rt_raw.calitp_files
[2022-04-11 21:33:43,805] {pod_launcher.py:149} INFO -          because it is a source
[2022-04-11 21:33:46,031] {pod_launcher.py:198} INFO - Event: dbt-run-and-upload-artifacts.ca426ecbd8a24fd0ba46f01878f26806 had an event of type Succeeded
```

### CODE REVIEW

File: warehouse/scripts/push_to_metabase.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+dbt-metabase models \
+    --dbt_manifest_path ./target/manifest.json \
+    --dbt_database cal-itp-data-infra \
+    --metabase_host dashboards.calitp.org \
+    --metabase_user $METABASE_USER \
+    --metabase_password $METABASE_PASSWORD \
+    --metabase_database "(Internal) Staging Warehouse Views"
```
Comment (lauriemerrell) [inline warehouse/scripts/push_to_metabase.sh:10]: Do we want this hard coded? I guess this is for testing?

**Review by lauriemerrell — COMMENTED** [review]
To clarify, the doc block TODO is in the forked repo right? Not to be added here.

File: warehouse/scripts/push_to_metabase.sh (hunk 1 above)
Comment (atvaccaro) [reply warehouse/scripts/push_to_metabase.sh:10]: Oh yeah this was just for testing, it's not that useful actually. Will delete to avoid confusion.

**Comment by lauriemerrell** [comment]
I'm not so familiar with the etiquette around forks, do we want to add comments to the README over in the fork to explain what we've added/changed? And do we want to be doing PR reviews in that fork going forward?

**Comment by atvaccaro** [comment]
> To clarify, the doc block TODO is in the forked repo right? Not to be added here.

I may have spoke too soon, and I just messed up what I was doing. The project can either parse dbt YAML or a manifest.json; I originally switched to using the manifest.json but hadn't rebuilt the image to point at the proper Metabase database. So I will strike that TODO as it's working now.

image

**Comment by atvaccaro** [comment]
> I'm not so familiar with the etiquette around forks, do we want to add comments to the README over in the fork to explain what we've added/changed? And do we want to be doing PR reviews in that fork going forward?

I plan to open a PR against the upstream repo when possible; these are features that would be nice to have in the main project and the maintainer is fairly active.

**Review by lauriemerrell — APPROVED** [review]

**Comment by blakeflei** [comment]
> > I'm not so familiar with the etiquette around forks, do we want to add comments to the README over in the fork to explain what we've added/changed? And do we want to be doing PR reviews in that fork going forward?
> 
> I plan to open a PR against the upstream repo when possible; these are features that would be nice to have in the main project and the maintainer is fairly active.

Do we have this tracked? Is there is urgency for us doing it this way?

**Comment by atvaccaro** [comment]
> Do we have this tracked? Is there urgency or some other reason for us forking our own repo vs just making PRs for the upstream repo?

I think we discussed in Slack but I'll open up a PR against the upstream now; in terms of urgency, yes we need these fixes because we don't import sources into Metabase (without my changes, the sync fails), and we want to be able to specify proper display names since our model names use so many acronyms (specifically GTFS) that are incorrectly being title-cased.

**Comment by atvaccaro** [comment]
as promised: https://github.com/gouline/dbt-metabase/pull/95

**Pull request merged**